### DISCUSSION TITLE
prometheus alert and Alert manager are not running

### DISCUSSION THREAD

**Discussion by bharath-gaveni** [learndevopsonline/learndevopsonline#1640] [Q&A] [answered]
prometheus alert and Alert manager are not running

![Screenshot (27)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/fc457a41-043c-460f-813d-da78f583f06e)
![Screenshot (31)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/396d4733-1eca-44b3-b802-5447eb6cb081)

**Comment by sreedharm07** [comment]
Your prometheus server was in status inactive 
Please configure prometheus config properly

**Reply by bharath-gaveni** [reply]
every time we have to do systemctl start for prothemeus and alertmanager

**Reply by sreedharm07** [reply]
Yes
To not do every time do enable

**Reply by bharath-gaveni** [reply]
I have problem like after running ansible pull prometheus and alert manager are going to inactive state 
![Screenshot (30)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/016cfe7a-be4c-4ff0-b3a8-960eb7942e59)
![Screenshot (28)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/0333fc0c-8ea2-4ab1-86ed-058312a9906c)

**Reply by bharath-gaveni** [reply]
even after enabling its not working 
![Screenshot (33)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/3888edac-3080-4528-951d-e282375eb68a)

**Reply by sreedharm07** [reply, marked as answer]
Please configure prometheus config properly 
If you see systemctl status prometheus. your prometheus was in inactive state

So, check prometheus config correctly

**Reply by bharath-gaveni** [reply]
okay thank you @sreedharm07

**Reply by sreedharm07** [reply]
just now i checked its working now 
prometheus port is opened
